tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -2229,6 +2229,114 @@ TEST_F(OpConverterTest, ConvertSquare) {
   // TestConvertSquare<DT_HALF>(this);
 }
 
+#if NV_TENSORRT_MAJOR > 5 || (NV_TENSORRT_MAJOR == 5 && NV_TENSORRT_MINOR >= 1)
+TEST_F(OpConverterTest, ConvertCombinedNMS) {
+  {
+    // Input list is empty, should fail.
+    NodeDef node_def = MakeNodeDef("my_nms", "CombinedNonMaxSuppression", {});
+    RunValidationAndConversion(node_def, error::INVALID_ARGUMENT,
+                               "Six inputs expected for CombinedNonMaxSuppression, at my_nms");
+  }
+  // Get the NodeDef for CombinedNMS.
+//  auto get_nms_nodedef = []() -> NodeDef {
+//      Scope s = Scope::NewRootScope();
+//      auto boxes_tensor = ops::Placeholder(s.WithOpName("boxes"), DT_FLOAT);
+//      auto scores_tensor = ops::Placeholder(s.WithOpName("scores"), DT_FLOAT);
+//      auto max_output_size_per_class = ops::Placeholder(s.WithOpName("max_output_size_per_class"), DT_INT32);
+//      auto max_total_size = ops::Placeholder(s.WithOpName("max_total_size"), DT_INT32);
+//      auto iou_threshold = ops::Placeholder(s.WithOpName("iou_threshold"), DT_FLOAT);
+//      auto score_threshold = ops::Placeholder(s.WithOpName("score_threshold"), DT_FLOAT);
+//      auto nms_attrs = ops::CombinedNonMaxSuppression::Attrs().PadPerClass(false);
+//
+//      auto nms_op = ops::CombinedNonMaxSuppression(s.WithOpName("my_nms"),
+//                                                   boxes_tensor,
+//                                                   scores_tensor,
+//                                                   max_output_size_per_class,
+//                                                   max_total_size,
+//                                                   iou_threshold,
+//                                                   score_threshold,
+//                                                   nms_attrs);
+//      return nms_op.operation.node()->def();
```

Comment:
I am still working on this.

This is the line I get a crash. `node()` returns null pointer.

Reply:
This is super weird. I'll try probably next week to see what happens.

Reply:
Hi Pooya, I copied this portion of the code (without the actual tests below) but cannot reproduce the problem. Would you please uncomment all of them, and fix the formats? Then once RC1 is out I can patch and test again.

Thanks.